.github/workflows/deploy.yml

```diff
@@ -13,7 +13,7 @@ env:
 
 jobs:
   build-and-push-image:
-    if: ${{ github.ref == 'refs/heads/master' }}
+    if: ${{ github.ref == 'refs/heads/master' || github.ref == 'refs/heads/smtp-from' }}
```

Comment:
@tmfrnz if you want we can remove this check and allow push from any branch?

Reply:
done =)